core/field_angle.js

```diff
@@ -70,51 +107,87 @@ Blockly.FieldAngle.fromJson = function(options) {
  */
 Blockly.FieldAngle.prototype.SERIALIZABLE = true;
 
-/**
- * Round angles to the nearest 15 degrees when using mouse.
- * Set to 0 to disable rounding.
- */
-Blockly.FieldAngle.ROUND = 15;
-
 /**
  * Half the width of protractor image.
+ * @const {number}
  */
 Blockly.FieldAngle.HALF = 100 / 2;
 
-/* The following two settings work together to set the behaviour of the angle
- * picker.  While many combinations are possible, two modes are typical:
- * Math mode.
- *   0 deg is right, 90 is up.  This is the style used by protractors.
- *   Blockly.FieldAngle.CLOCKWISE = false;
- *   Blockly.FieldAngle.OFFSET = 0;
- * Compass mode.
- *   0 deg is up, 90 is right.  This is the style used by maps.
- *   Blockly.FieldAngle.CLOCKWISE = true;
- *   Blockly.FieldAngle.OFFSET = 90;
+/**
+ * Radius of protractor circle.  Slightly smaller than protractor size since
+ * otherwise SVG crops off half the border at the edges.
+ * @const {number}
  */
+Blockly.FieldAngle.RADIUS = Blockly.FieldAngle.HALF - 1;
 
 /**
- * Angle increases clockwise (true) or counterclockwise (false).
+ * Default property describing which direction makes an angle field's value
+ * increase. Angle increases clockwise (true) or counterclockwise (false).
+ * @const {boolean}
  */
 Blockly.FieldAngle.CLOCKWISE = false;
 
 /**
- * Offset the location of 0 degrees (and all angles) by a constant.
+ * The default offset of 0 degrees (and all angles). Always offsets in the
+ * counterclockwise direction, regardless of the field's clockwise property.
  * Usually either 0 (0 = right) or 90 (0 = up).
+ * @const {number}
  */
 Blockly.FieldAngle.OFFSET = 0;
 
 /**
- * Maximum allowed angle before wrapping.
+ * The default maximum angle to allow before wrapping.
  * Usually either 360 (for 0 to 359.9) or 180 (for -179.9 to 180).
+ * @const {number}
  */
 Blockly.FieldAngle.WRAP = 360;
 
 /**
- * Radius of protractor circle.  Slightly smaller than protractor size since
- * otherwise SVG crops off half the border at the edges.
+ * The default amount to round angles to when using a mouse or keyboard nav
+ * input. Must be a positive integer to support keyboard navigation.
+ * @const {number}
  */
-Blockly.FieldAngle.RADIUS = Blockly.FieldAngle.HALF - 1;
+Blockly.FieldAngle.ROUND = 15;
```

Comment:
Why the re-order?

Reply:
It feels better.
Half, Radius, Clockwise, Offset, Wrap, Round vs
Round, Half, Clockwise, Offset, Wrap, Radius

I didn't think about it causing merge conflicts though, so I can change it back.